plugins/chat/assets/javascripts/discourse/modifiers/chat/resizable-node.js

```diff
@@ -137,7 +160,31 @@ export default class ResizableNode extends Modifier {
       Object.assign(this.element.style, newStyle);
     }
 
-    this.didResizeContainer?.(this.element, { width, height });
+    this.didResizeContainer?.(this.element, {
+      width: width >= this._minimumWidth ? width : this._minimumWidth,
+      height: height >= this._minimumHeight ? height : this._minimumHeight,
+    });
```

Comment:
This was actually a bug where we would report a width which we haven't actually set, this wouldn't be visible as we are saved by the css min-width but was still incorrect and could create weird states.